xptifw/unit_test/xpti_api_tests.cpp

```diff
@@ -201,39 +212,31 @@ TEST(xptiApiTest, xptiTraceEnabled) {
   EXPECT_EQ(Result, false);
 }
 
-XPTI_CALLBACK_API void trace_point_callback(uint16_t trace_type,
-                                            xpti::trace_event_data_t *parent,
-                                            xpti::trace_event_data_t *event,
-                                            uint64_t instance,
-                                            const void *user_data) {
-
-  if (user_data)
-    (*(int *)user_data) = 1;
-}
+void trace_point_callback(uint16_t /*trace_type*/,
+                          xpti::trace_event_data_t * /*parent*/,
+                          xpti::trace_event_data_t * /*event*/,
+                          uint64_t /*instance*/, const void * /*user_data*/) {}
```

Comment:
Why are you removing the body of this callback?

Reply:
In the context where it is used, it is only used as a function pointer of particular type, i.e. it is never called.